src/platform/Linux/NetworkCommissioningThreadDriver.cpp

```diff
@@ -197,6 +197,25 @@ bool LinuxThreadDriver::ThreadNetworkIterator::Next(Network & item)
     return true;
 }
 
+ThreadCapabilities LinuxThreadDriver::GetSupportedThreadFeatures()
+{
+    BitMask<ThreadCapabilities> capabilites = 0;
+    capabilites.SetField(ThreadCapabilities::kIsBorderRouterCapable,
+                         CHIP_DEVICE_CONFIG_THREAD_BORDER_ROUTER /*OPENTHREAD_CONFIG_BORDER_ROUTER_ENABLE*/);
```

Comment:
what is the meaning of the comment?

Reply:
I'll write a clearer comment. The goal is that `CHIP_DEVICE_CONFIG_THREAD_BORDER_ROUTER` should either set the thread stack define (`OPENTHREAD_CONFIG_BORDER_ROUTER_ENABLE`) or at least ensure it is the same value as we don't have access to the thread stack define in matter sdk